contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -111,6 +117,9 @@ def span_data_to_envelope(self, sd):
                 data.type = 'INPROC'
         # TODO: links, tracestate, tags
         for key in sd.attributes:
+            # This removes redundant data from AppInsights
```

Comment:
Minor, `AppInsights`->`ApplicationInsights` since [AppInsights](https://www.appdirect.com/products/appinsights) is a different thing.

Reply:
Fixed!